index.html

```diff
@@ -506,6 +507,34 @@ <h2>Safe usage</h2>
       <code>defer</code>) but a site could fix any misguesses across the board by being explicit using Priority Hints.</p>
   </section>
 
+  <section>
+    <h2>Security and Privacy Considerations</h2>
+    <h3>Fingerprintability</h3>
+    <p>This specification does not expose any new information about the user and their preferences, so does not increase the risk of user fingerprinting.
+    At the same time, it does expose priority information to web servers that enables them to distinguish between browsers that support the feature and ones that do not support it.
+    This is not dissimilar to other web platform features, but with one important distinction: 
+    this information is exposed at the HTTP layer, allowing the server implementation to discover discrepancies between the User-Agent string the request is sent with and the browser that sent the request. 
+    It is not clear that this is a significant issue, but it is something for implementing user agents (and user agents which try to spoof others) to be aware of.
```

Comment:
@lknik - Added a sentence calling out network observers specifically, let me know if it covers your concerns

Reply:
you mean like, passive network observers?

Reply:
yeah (the sentence below)

Reply:
I'm not sure if webservers cannot observe if the browser supports the feature or not (different priorities used), but it is not the main concern, I guess. Not sure we need the discussion on UA - up to you. Basically it's whether it makes sense to document that passive attackers maybe can spot differences (https/tor-like scenarios). I'm not particularly concerned with fingerprinting beyond that.

Reply:
> I'm not sure if webservers cannot observe if the browser supports the feature or not

I believe you're saying you think webservers _may_ be able to determine whether a UA supports the feature. I think that is currently what Yoav mentioned in the spec, no? Either way, any change in priority doesn't necessarily mean a UA supports this particular feature. Chrome, for example, runs all kinds of experiments that may touch the priorities of certain resources, for a certain percentage of users etc.

Reply:
But as I said it's not the main concern, (1/0 supported or not).